muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/locales/translations/fi.json

```diff
@@ -2120,5 +2120,18 @@
     "notifications": {
       "signUpError": "Opiskelijan ilmoittaminen kurssille epäonnistui"
     }
+  },
+  "courseCarousel": {
```

Comment:
For now this seems ok, however this need to be monitored and maybe refactored in the future as I don't see any reason for course carousel to have it's own namespace but this might change.

Everything here is quite mundane and basic strings that could be used within common namespace. `checkOut` and `signUp` are located under workspace namespace currently but they should be moved to common namespace as we are using those string at least in 2 locations.

`empty` also could be placed under common and extended with necessary context ie. `empty_course-carousel`.

If we display this  `guidanceCouncelorSuggestion` outrside of course carousel it can be also relocated to common namespace.

Alternatively all of these could be located under workspace namespace as they are specifically targeted to workspace related views. This might be the best scenario in the end.